### PR TITLE
Added boost license as part of moveit_simple's package.xml for pretty…

### PR DESCRIPTION
**Pull request by geoffreychiou**
…print

### CODE REVIEW

File: moveit_simple/package.xml [hunk 1]
```diff
@@ -3,8 +3,15 @@
   <name>moveit_simple</name>
   <version>0.1.0</version>
   <description>Includes simple wrappers for calling MoveIt as a ROS based library.  Similar function as MoveGroup but all calls are made locally using direct MoveIt objects.  This library assumes that MoveIt is being used in a broader ROS system (i.e. certain parameters, like robot_description exist)</description>
+  
   <maintainer email="Edwards@todo.todo">Shaun Edwards</maintainer>
```
Comment (130s) [inline moveit_simple/package.xml]: Awesome email btw.

**Review by 130s — APPROVED** [review]

File: moveit_simple/package.xml (hunk 1 above)
Comment (geoffreychiou) [reply moveit_simple/package.xml]: Fixed.

**Comment by geoffreychiou** [comment]
@130s I added some changes since your last review.

File: moveit_simple/package.xml [hunk 2]
```diff
@@ -3,8 +3,12 @@
   <name>moveit_simple</name>
   <version>0.1.0</version>
   <description>Includes simple wrappers for calling MoveIt as a ROS based library.  Similar function as MoveGroup but all calls are made locally using direct MoveIt objects.  This library assumes that MoveIt is being used in a broader ROS system (i.e. certain parameters, like robot_description exist)</description>
-  <maintainer email="Edwards@todo.todo">Shaun Edwards</maintainer>
-  <license>Apache 2.0</license>
+  
+  <maintainer email="shaun.edwards@gmail.com">Shaun Edwards</maintainer>
+  
+  <!-- moveit_simple, prettyprint -->
+  <license>Apache 2.0, Boost Software License</license>
```
Comment (130s) [inline moveit_simple/package.xml:10]: Putting multiple licenses is not what license element is supposed to be defined.

Comment (geoffreychiou) [reply moveit_simple/package.xml:10]: Yea, should I revert or just submit a new one fixing?

Comment (130s) [reply moveit_simple/package.xml:10]: Whichever more punishing for you :)

Comment (geoffreychiou) [reply moveit_simple/package.xml:10]: Fixed in https://github.com/plusone-robotics/moveit_simple/pull/58. :stuck_out_tongue:

**Pull request merged**